From acac58dab677eb1e93d4dfdfbcc2d604fa83ce01 Mon Sep 17 00:00:00 2001
From: orangepi <buddy.d.zhang@gmail.com>
Date: Fri, 7 Apr 2017 17:20:23 +0800
Subject: [PATCH] [PATCH] Uboot for Mali450

---
 include/configs/sun50iw2p1.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/sun50iw2p1.h b/include/configs/sun50iw2p1.h
index a44689f..410d956 100755
--- a/include/configs/sun50iw2p1.h
+++ b/include/configs/sun50iw2p1.h
@@ -299,6 +299,7 @@
 	"script=boot.scr\0" \
 	"mmcboot=run load_dtb load_kernel load_initrd set_cmdline boot_kernel\0" \
 	"console=ttyS0,115200\0" \
+	"cma=300M\0" \
 	"root=/dev/mmcblk0p2\0" \
 	"load_addr=40000000\0" \
 	"fdt_addr=44000000\0" \
-- 
1.9.1

